src/form_builder/views.py

```diff
@@ -173,6 +173,8 @@ def respond(req, id):
     user_form = get_object_or_404(Form, slug=id)
     already_responded = AnonymousResponse.objects.check_dupe(user_form.id,
                                                              req.user.username)
+    is_closed = user_form.is_closed()
```

Comment:
It might be. I moved it up here because I wasn't sure it was working down there, but I think what fixed it was adding parentheses after moving it up here. So I'll try it with the parentheses down there.